manifests/m/Mozilla/Firefox/109.0.1/Mozilla.Firefox.installer.yaml

```
@@ -0,0 +1,690 @@
+  InstallerSha256: 82FCC46B7513A6D04A343A10D0D9F0BE90D97A25F5690691532CCB0568A76222
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  AppsAndFeaturesEntries:
+  - DisplayName: Mozilla Firefox (x86 en-GB)
+    Publisher: Mozilla
+    ProductCode: Mozilla Firefox 109.0.1 (x86 en-GB)
+- InstallerLocale: en-GB
+  Architecture: arm64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win64-aarch64/en-GB/Firefox%20Setup%20109.0.1.exe
+  InstallerSha256: 0F5AE2ABBDE2CD197F86493BD8CBD455519AF37814104814B7D635275BE4C2E7
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+- InstallerLocale: ru-RU
+  Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win64/ru/Firefox%20Setup%20109.0.1.msi
+  InstallerSha256: E9469F202E6CE11EC4CCCCB634C66795EFC640F0DA5E8D71175136FB612EDA5E
+  AppsAndFeaturesEntries:
+  - DisplayName: Mozilla Firefox (x64 ru)
+    Publisher: Mozilla
+    ProductCode: Mozilla Firefox 109.0.1 (x64 ru)
+- InstallerLocale: ru-RU
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win64/ru/Firefox%20Setup%20109.0.1.exe
+  InstallerSha256: 33964479E5DB8B07EB9F3A0DBC56EF6DF0567243EDF4B89AC543E5D8F5E76914
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  AppsAndFeaturesEntries:
+  - DisplayName: Mozilla Firefox (x64 ru)
+    Publisher: Mozilla
+    ProductCode: Mozilla Firefox 109.0.1 (x64 ru)
+- InstallerLocale: ru-RU
+  Architecture: x86
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win32/ru/Firefox%20Setup%20109.0.1.msi
+  InstallerSha256: 65BD07426CBA8CDE1B3C0EDCADF122EA59EC8BCAB09F3D5100C42F13BD8F33DD
+  AppsAndFeaturesEntries:
+  - DisplayName: Mozilla Firefox (x86 ru)
+    Publisher: Mozilla
+    ProductCode: Mozilla Firefox 109.0.1 (x86 ru)
+- InstallerLocale: ru-RU
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win32/ru/Firefox%20Setup%20109.0.1.exe
+  InstallerSha256: 79C8C6489223AB7410E9401AE5A3CAB8D9CA6B078563329A98469400BAB6E8E7
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+  AppsAndFeaturesEntries:
+  - DisplayName: Mozilla Firefox (x86 ru)
+    Publisher: Mozilla
+    ProductCode: Mozilla Firefox 109.0.1 (x86 ru)
+- InstallerLocale: ru-RU
+  Architecture: arm64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win64-aarch64/ru/Firefox%20Setup%20109.0.1.exe
+  InstallerSha256: 74E060F1570A485385ABA1170AE6E54BFA9A40DA3A3E73C62E720FD5AA29EA4D
```

Hi @jedieaston, this is the correct hash for the `https://download-installer.cdn.mozilla.net/pub/firefox/releases/109.0.1/win64-aarch64/ru/Firefox%20Setup%20109.0.1.exe` URL.
```suggestion
  InstallerSha256: 18F48AC06B006EF9F071267E91E10A8EBECA74397821C8CCE3FFAE9D6BDAEC0A
```